modules/auxiliary/scanner/wproxy/att_open_proxy.py

```diff
@@ -1,7 +1,13 @@
 #!/usr/bin/env python3
 
-from metasploit import module, probe_scanner
+import sys
+from metasploit import module
 
+dependencies_missing = False
+if sys.version_info >= (3, 8):
```

Comment:
This likely belongs in the check itself, but as there's only one module using this API - I'll skip it for now